jest.config.js

```diff
@@ -86,7 +86,7 @@ module.exports = {
 
   // A map from regular expressions to module names or to arrays of module names that
   // allow to stub out resources with a single module
-  // moduleNameMapper: {},
+  moduleNameMapper: { '\\.(css|less)$': '<rootDir>/styleMock.js' },
```

Comment:
does this get removed later or is this the workaround? Seems a little hacky. 🤔

Reply:
this is official documentation, https://jestjs.io/docs/webpack